docs/LOGGING.md

```diff
@@ -1,6 +1,6 @@
 # EVE LOGGING PIPELINE
 
-This document describes the different stages logs on a EVE edge node go through before being delivered to cloud. It also describes the various congestion points in the current log pipeline along with the places where logs could leak (dropped on the floor) with the current state of EVE code.
+This document describes the different stages logs on a EVE edge node go through before being delivered to cloud. It also describes the various congestion points in the current log pipeline along with the places where logs could leak (dropped on the floor). It also describes when and where log entries can be lost.
```

Comment:
The second sentence in this paragraph is no longer needed given the third sentence you added.

Reply:
will remove.